src/main/resources/assets/botania/lang/zh_CN.lang

```diff
@@ -3682,13 +3754,16 @@ botania.page.stoneAlchemy0=用&1炼金催化器&0就能做出新类型 的&1石
 botania.page.stoneAlchemy1=制作不同的石头
 botania.page.stoneAlchemy2=磨光
 botania.page.stoneAlchemy3=砌砖
-botania.page.stoneAlchemy4=錾制
+
+# Not sure why these are not removed but I will remove them in zh_CN at least
+# Not sure why these are not removed but I will remove them in zh_CN at least
+# Not sure why these are not removed but I will remove them in zh_CN at least
 
```

Comment:
For [line 3758 - line 3760](https://github.com/williewillus/Botania/blob/master/src/main/resources/assets/botania/lang/en_US.lang#L3758), I think they are legacy remain, so I replaced them with some lorem ipsum thingy instead.